packages/client/src/constants/maps.js

```diff
@@ -14,20 +14,23 @@ export const COIN_TYPE_TO_META = {
     displayName: "FLOW",
     vaultName: "FlowToken.Vault",
     publicReceiverPath: { domain: "public", identifier: "flowTokenReceiver" },
+    publicBalancePath: { domain: "public", identifier: "flowTokenBalance" },
     storageVaultPath: { domain: "storage", identifier: "flowTokenVault" },
   },
   [COIN_TYPES.FUSD]: {
     contractName: "FUSD",
     displayName: "FUSD",
     vaultName: "FUSD.Vault",
     publicReceiverPath: { domain: "public", identifier: "fusdReceiver" },
+    publicBalancePath: { domain: "public", identifier: "fusdBalance" },
     storageVaultPath: { domain: "storage", identifier: "fusdVault" },
   },
   [COIN_TYPES.USDC]: {
     contractName: "FiatToken",
     displayName: "USDC",
     vaultName: "FiatToken.Vault",
     publicReceiverPath: { domain: "public", identifier: "USDCVaultReceiver" },
+    publicBalancePath: { domain: "public", identifier: "USDCVaultBalance" },
```

Comment:
Can we add this field for the rest of the token too? Looks like `storageVaultPath` also needs to be added there. 
Until there is better ways to retrieve this metadata, we'll have to maintain this list to ensure all supported tokens has the required metadata here, or else flow will break for that token type. 
https://github.com/portto/blt-contracts/blob/e89b1af29793eb194f541a0c9a1e1adca590aaad/contracts/flow/token/BloctoToken.cdc#L189
https://github.com/StarlyIO/starly-token-contracts/blob/master/contracts/flow/contracts/StarlyToken.cdc#L107
https://flowscan.org/contract/A.d01e482eb680ec9f.REVV/overview